src/content/en/2020/mobile-web.md

```diff
@@ -23,12 +23,12 @@ queries: 12_Mobile_Web
 unedited: true
 ---
 
-## Introduction
+# Introduction
```

Comment:
The chapter title is the H1 on the page, so all of the subsequent headings should start at H2
```suggestion
## Introduction
```

Reply:
Ok I hope the fifth level down still looks like a heading ..

Reply:
5th is good (but only had to add that this year!). 6th is definitely getting into plain text territory! 😁 But we have that too if really needed.